tor/validation/formatting_validation.py

```diff
@@ -149,6 +153,24 @@ def check_for_fenced_code_block(transcription: str) -> Optional[FormattingIssue]
     )
 
 
+def check_for_autogenerated_long_hyphen(transcription: str) -> Optional[FormattingIssue]:
+    """Check if the transcription contains autogenerated long hyphens as a result of mobile device 'assistance'
+
+    Separator should look like this:
+    ---
+
+    Mobile devices may convert the separator to:
+    —- or -—
+
+    These don't display correctly on all devices
+    """
+    return (
+        FormattingIssue.AUTOGENERATED_LONG_HYPHEN
+        if AUTOGENERATED_LONG_HYPHEN_PATTERN.search(transcription) is not None

```

Comment:
Why not drop the regex and do a simple `x in y` check?

```suggestion
        if "\u2014" in transcription  # \u2014 == em dash == "long hyphen"

```